src/components/ClickCatcher/ClickCatcher.js

```diff
@@ -19,6 +20,13 @@ export default function ClickCatcher ({ tag, handleMouseOver, navigate, ...props
     if (node.getAttribute('target') !== '_blank') {
       node.setAttribute('target', '_blank')
     }
+
+    const matches = [...node.getAttribute('href').matchAll(hyloUrlRegex)]
+    if (matches[0] && matches[0].length === 2) {
+      node.setAttribute('target', '_self')
+      const urlPath = matches[0][1] === '' ? '/' : matches[0][1]
+      node.setAttribute('href', urlPath)
```

Comment:
for any hylo.com links, this link just cuts off the host altogether. So the `href` is just the path and will just hit that path on whatever host (local, staging or prod) the application is running on.

Anticipating that this will work on local, staging and prod